invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -9,11 +9,15 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, id='preview-iframe', width='100%', height='400' ) %}
+{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, include_deleted, id='preview-iframe', width='100%', height='400' ) %}
```

Comment:
could you check if we need to update templates for zenodo and cds?

Reply:
yes, here the PRs:
- https://github.com/zenodo/zenodo-rdm/pull/534
- https://github.com/CERNDocumentServer/cds-rdm/pull/106